packages/codemod/.claude/skills/codemod-patterns/SKILL.md

```
@@ -0,0 +1,607 @@
+
+Use Shared Utilities
+
+GOOD Uses utility 1 pass
+import { trackClassInstances, renameMethods } from '../lib/utils';
+const instances = trackClassInstances(j, root, 'Agent');
+if (instances.size === 0) return;
+renameMethods(j, root, instances, { oldMethod: 'newMethod' });
+
+BAD Manual implementation 2 passes
+const instances = new Set<string>();
+root.find(j.NewExpression).forEach(/_ ... _/);
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)